### PR TITLE
Clean docs

### PR DESCRIPTION
**Pull request by pitmonticone**

### CODE REVIEW

**Comment by pitmonticone** [comment]
Hi @javierm, thank you for starting the review of this PR! 

Since you assigned me I'd like to ask you if I'm supposed to do something else to help.

**Comment by javierm** [comment]
Hi, @pitmonticone :smile:.

Sorry! No, this is just so your avatar appears in our [Kanban project](https://github.com/orgs/consul/projects/1?fullscreen=true). This is automatically done on pull request opened in [consul](https://github.com/consul/consul/) or [the installer](https://github.com/consul/installer/) but we haven't automated it for this repository yet.

We'll review this pull request when we can, though unfortunately the amount of pull requests pending to be reviewed has grown a bit in the last couple of months. Until we review it, no need for you to do anything :wink:.

Thanks for the pull request, by the way!

**Comment by pitmonticone** [comment]
Got it. Thanks!

File: en/features/graphql.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
-* [Examples of queries](#examples-of-queries)
-  * [Request a single record from a collection](#request-a-single-record-from-a-collection)
-  * [Request a complete collection](#request-a-complete-collection)
-    * [Pagination](#pagination)
-  * [Accessing several resources in a single request](#accessing-several-resources-in-a-single-request)
-* [Security limitations](#security-limitations)
-  * [Example of too deep query](#example-of-too-deep-query)
-  * [Example of too complex query](#example-of-too-complex-query)
-* [Code examples](#code-examples)
+- [API Documentation](#api-documentation)
```
Comment (taitus) [inline en/features/graphql.md]: I am not sure if we need this line 🤔 . 
If we remove it this page is still displayed the same in both English and Spanish.

File: en/getting_started/create.md [hunk 2]
```diff
@@ -2,15 +2,15 @@
 
 Consul git repo is hosted at Github.com, we recommend using it for your fork's repo to make things easier. But you can use any other service like Bitbucket or Gitlab if you want to, just don't forget to put a reference link back to CONSUL on the footer to comply with project's license (GPL Affero 3).
 
-1. [Register an user account on Github](https://github.com/join) if you don't have one
+1. [Register an user account on Github](https://github.com/join) if you don't have one.
 
-2. [Create an Organization](https://help.github.com/articles/creating-a-new-organization-from-scratch/) on Github with the name of your city or the organization that's going to use Consul. **This is not mandatory**, but it will help undertand the fork's purpose and future contributions by other users.
+2. [Create an Organization](https://help.github.com/articles/creating-a-new-organization-from-scratch/) on Github with the name of your city or the organization that's going to use Consul. **This is not mandatory**, but it will help underhand the fork's purpose and future contributions by other users.
```
Comment (taitus) [inline en/getting_started/create.md]: Perhaps the original text meant to try to say "understand". 
I don't know if `underhand` makes more sense than `understand` in this case 🤔

File: es/customization/customization.md [hunk 3]
```diff
@@ -8,4 +8,4 @@
 * [Modelos](models.md)
 * [Componentes](components.md)
 * [Gemas](gems.md)
-* [Adaptar la aplicación](overwritting.md)
+* [Adaptar la aplicación](overwriting.md)
```
Comment (taitus) [inline es/customization/customization.md:11]: With this change, the name of the Spanish file would need to be updated from `overwritting.md` to `overwriting.md`. The same as has been done for the English file.

**Review by taitus — COMMENTED** [review]
Hi @pitmonticone 

Thank you very much for the PR 👏  and sorry for the delay in the review.

Here are a couple of comments about the PR, see what you think. If you think they are correct, could you apply them? Thank you very much!

Greetings!

File: en/features/graphql.md (hunk 1 above)
Comment (pitmonticone) [reply en/features/graphql.md]: Ops, sorry. You're right, it shouldn't be there. I'll change it.

File: en/getting_started/create.md (hunk 2 above)
Comment (pitmonticone) [reply en/getting_started/create.md]: No, it was my mistake. I'll fix it.

File: es/customization/customization.md (hunk 3 above)
Comment (pitmonticone) [reply es/customization/customization.md:11]: I'll do it.

**Comment by pitmonticone** [comment]
> Here are a couple of comments about the PR, see what you think. If you think they are correct, could you apply them?

Thank you @taitus for your review. I've just finished applying the requested changes. 

May I ask what's required to join this GitHub organisation? Thank you.

**Review by taitus — APPROVED** [review]

**Comment by taitus** [comment]
@pitmonticone Thank you for adding the suggested changes 👏 . 

> May I ask what's required to join this GitHub organisation? Thank you.

I have just sent you an invitation 👌 

If you are interested Consul have a slack channel. You can request access at the following link: https://consulproject.org/en/#contact

Greetings!

**Comment by pitmonticone** [comment]
Thank you very much @taitus.

**Comment by pitmonticone** [comment]
Thank you @taitus for inviting me as a collaborator. 

I should have asked: "What's required to join this GitHub organization as a member?".

Thanks in advance.

File: en/customization/introduction.md [hunk 4]
```diff
@@ -129,15 +129,15 @@ The aim of this feature is to allow users the introduction of dynamic contents i
 To activate this feature you must follow 2 steps:
 1. Execute the following command `bin/rake settings:create_translation_interface_setting RAILS_ENV=production` (This is only required for already existing intallations, for new consul installations this step is not needed).
 2. Accessing as administrator user to the administration panel of your Consul application to the section **Configuration > Features** and activating the feature called **Translation Interface** as you can see next:
-![Active interface translations](../../img/translations/interface_translations/active-interface-translations-en.png)
+![Active interface translations](../../img/translations/interface_translations/active-interface-translations-en.png).
```
Comment (javierm) [inline en/customization/introduction.md]: Since this is an image, I think we can omit the `.` here; otherwise it'd look strange :thinking:.

File: en/customization/views_and_styles.md [hunk 5]
```diff
@@ -26,6 +26,6 @@ Also you can check your scss files syntax with
 scss-lint
 ```
 
-## Accesibility
+## Accessibility
 
-To maintain accesibility level, if you add new colors use a [Color contrast checker](http://webaim.org/resources/contrastchecker/) (WCAG AA is mandatory, WCAG AAA is recommended)
+To maintain accessibility level, if you add new colors use a [Color contrast checker](http://webaim.org/resources/contrastchecker/) (WCAG AA is mandatory, WCAG AAA is recommended)
```
Comment (javierm) [inline en/customization/views_and_styles.md]: Since we're changing this line, does it make sense to add a `.` here?

```suggestion
To maintain accessibility level, if you add new colors use a [Color contrast checker](http://webaim.org/resources/contrastchecker/) (WCAG AA is mandatory, WCAG AAA is recommended).
```

File: en/features/graphql.md [hunk 6]
```diff
@@ -1,22 +1,22 @@
...
+- [Available information:](#available-information)
+- [Examples of queries](#examples-of-queries)
+  - [Request a single record from a collection](#request-a-single-record-from-a-collection)
+  - [Request a complete collection](#request-a-complete-collection)
+    - [Pagination](#pagination)
+  - [Accessing several resources in a single request](#accessing-several-resources-in-a-single-request)
+- [Security limitations](#security-limitations)
+  - [Example of too deep query](#example-of-too-deep-query)
+  - [Example of too complex query](#example-of-too-complex-query)
+- [Code examples](#code-examples)
```
Comment (javierm) [inline en/features/graphql.md]: I think we use `*` everywhere else to display a list of links :thinking:. Why the change here?

**Review by javierm — COMMENTED** [review]
@pitmonticone Thank you sooooo much for this contribution.

I've left a couple of comments. Let me know what you think!

File: en/customization/introduction.md (hunk 4 above)
Comment (pitmonticone) [reply en/customization/introduction.md]: Ops, sorry. Totally agree with you here! 

I'll change it as soon as I can.

File: en/customization/views_and_styles.md (hunk 5 above)
Comment (pitmonticone) [reply en/customization/views_and_styles.md]: Yes, I believe so. Thanks. 

I'm going to push the commit.

File: en/features/graphql.md (hunk 6 above)
Comment (pitmonticone) [reply en/features/graphql.md]: I didn't change it actually: VS Code convert it in that way when I save some changes in files containing tables of contents. Very strange behavior! 

I'll change it as soon as I can directly from the browser to avoid that problem.

**Comment by pitmonticone** [comment]
Hi @javierm, thank you very much for your comments. I've just applied all the necessary changes / corrections.

**Review by pitmonticone — COMMENTED** [review]
Hi @javierm, thank you very much for your comments. I've just applied all the necessary changes / corrections. 

May I ask what's required to join this GitHub organisation as a member? I'm very interested in deliberative / participatory democratic methods and tools. Thanks in advance.

**Review by javierm — APPROVED** [review]
@pitmonticone Awesome, thanks! :tada:

**Comment by javierm** [comment]
> I should have asked: "What's required to join this GitHub organization as a member?".

Thanks for your interest in contributing! :heart: If you'd like to collaborate more closely with the project, please [contact the Consul Democracy Foundation](https://consulproject.org/en/#contact). It would be awesome if you could include some details about what you're using Consul for.

Thanks a lot!
